apps/modernization-ui/src/apps/patient/add/extended/AddPatientExtended.tsx

```diff
@@ -11,12 +11,20 @@ import { AddExtendedPatientInteractionProvider } from './useAddExtendedPatientIn
 import { AddPatientExtendedNav } from './nav/AddPatientExtendedNav';
 
 import styles from './add-patient-extended.module.scss';
+
+import { ChangeEvent, useState } from 'react';
+import { Confirmation } from 'design-system/modal';
+import { Checkbox } from '@trussworks/react-uswds';
+import { useLocalStorage } from 'storage';
 import { Shown } from 'conditional-render';
 import { PatientCreatedPanel } from '../PatientCreatedPanel';
 import { CreatedPatient } from './api';
 
 export const AddPatientExtended = () => {
     const interaction = useAddExtendedPatient({ transformer, creator });
+    const [cancelModal, setCancelModal] = useState<boolean>(false);
+    const [visibilityCheckBox, setVisibilityCheckBox] = useState<boolean>(false);
```

Comment:
suggestion:   The `visibilityCheckBox` state is only used by the confirmation modal, you could create a `CancelAddPatientExtendedPanel` component that could encapsulate the logic for cancel

Reply:
suggestion: The name `visibilityCheckBox` doesn't really express what that actually is, it's more `doNotShowAgain`